tests/cases/conformance/jsdoc/typedefCrossModule5.ts

```diff
@@ -0,0 +1,12 @@
+// @pretty: true

```

Comment:
can we just log the additional info to the .errors baseline. it is nice that we have `--pretty` tests, but do not think all of the tests should be `--pretty` tests. they are hard to read really in baseline format.

Reply:
it is also good to see how this change affects our existing tests too.

Reply:
I should now be recording related info in the error baselines.